### ISSUE TITLE
StepperRow Image / Text overlap

### ISSUE THREAD

**Issue by nidegen** [xmartlabs/Eureka#1838]
- Environment: Eureka 5.0.0, Xcode 10.2 and iOS 12.2 

Problem: When adding an image to the stepper row cell, the title overlaps the image.

Stringely, when the form updates with `evaluateHidden()` of another section, the title jumps to the right place, only to move back when using the stepper button..

Is that intentional? I can have a look for a fix if not.
```
<<< StepperRow() { row in
      row.title = "Title"
      row.displayValueFor = { value in
        guard let value = value else { return nil }
        return "\(Int(value))"
      }
    }.cellSetup { cell, row in
      cell.imageView?.image = UIImage(named: "image_name")
      }
```

**Comment by mats-claassen** [comment]
Could you post a screenshot? However from what you say it does seem to be an issue. The text should never overlap with the image. Must be an unhandled case in the constraints. Fixes are welcome

**Comment by nidegen** [comment]
![a](https://user-images.githubusercontent.com/8779423/58558139-551c6780-8220-11e9-8388-d15a9d588dea.png)
![t](https://user-images.githubusercontent.com/8779423/58558141-55b4fe00-8220-11e9-8a2f-bc80e3b1eca1.png)

This is how the two states look. Any tips or hints for the fix? Could it be a autolayout bug?

**Comment by mats-claassen** [comment]
It seems the `StepperRow` has no constraints for the `imageView`.
It should be structured more like `SliderRow`, for example, having constraints set for different cases (like having an image or not) and updating them in `updateConstraints`

**Comment by nidegen** [comment]
see #1857 1857

**Comment by nidegen** [comment]
solved in #1857
